tck/src/main/java/org/eclipse/microprofile/config/tck/ConfigPropertiesTest.java

```diff
@@ -191,6 +192,7 @@ public String getHost() {
         }
     }
 
+    @Dependent
```

Comment:
Just want to add a note to clarify this.. Sincer there is a beans.xml, all classes are beans. I guess this is for Quarkus to work since it relies on Bean Defining annotations to optimistation. I thought I put there in my original PR but was asked to be removed. It is ok to specify this.

Reply:
We removed it because BeanThree was only being used for the programmatic API, that we later removed :)

Reply:
ah, ok. I might have remembered wrong.